dhis-2/dhis-services/dhis-service-tracker/src/test/resources/tracker/event_events_and_enrollment.json

```diff
@@ -555,40 +421,6 @@
       ],
       "completedBy": "admin",
       "completedAt": "2019-01-28T00:00:00.000",
-      "dataValues": [
```

Comment:
why are you deleting all of these?

Reply:
Because they are not relevant for this test and before the validation was not running.
To maintain them here I would need to change a lot of things in the metadata to make them work